tooling/templatize/pkg/pipeline/arm.go

```diff
@@ -32,6 +33,45 @@ func newArmClient(subscriptionID, region string) *armClient {
 	}
 }
 
+func getDeployment(ctx context.Context, rgName, deploymentName string, client *armresources.DeploymentsClient) (*armresources.DeploymentsClientGetResponse, error) {
+	resp, err := client.Get(ctx, rgName, "asd", nil)
```

Comment:
```suggestion
	resp, err := client.Get(ctx, rgName, deploymentName, nil)
```
unless I'm missing something?